src/__tests__/element-queries.js

```diff
@@ -375,6 +383,22 @@ test('using jest helpers to check element class names', () => {
   ).toThrowError()
 })
 
+test('using jest helpers to check element role', () => {
+  const {getByRole} = render(`
+    <div role="dialog">
+      <span>Contents</span>
+    </div>
+  `)
+
+  expect(getByRole('dialog')).toBeTruthy()
```

Comment:
This assertion is unnecessary because the next assertion exists :)

Reply:
Removed assertions

I basically copied from other tests so I wasn't sure how bullet proof you wanted it.